atm.js

```diff
@@ -1 +1,122 @@
-// your code goes here!
+// variable: checkingAmount
+// variable: savingsAmount
+//Branched!
+
+var checkingAmount = 0;
+var savingsAmount = 0;
+
+function depositMoneyChecking() {
+// Deposit money adds into checkingAmount
+// side effect: adds to specific account
+// side effect: update display
+// return: account balance
+  var textBox = parseInt(document.querySelector("#checkingAmount").value);
+  checkingAmount += textBox;
+  el = document.querySelector("#checkingBalanceDiv");
+  el.innerHTML = ("$" + checkingAmount);
+  checkZero();
+  return checkingAmount;
```

Comment:
Both `checkingAmount` & `savingsAmount` are a level of scope (currently in global scope) above these functions, any changes the functions make to them will happen without having to return anything since they point to the same thing.